### PR TITLE
Fix/repo-4792 - Alfresco/transforms and alfresco/renditions can't read path-to-jar on Windows systems

### PR DESCRIPTION
**Pull request by KikoUK**
- Unit tests passed successfully with repo-4792-1 release version in acs-repository (travis) and acs-packaging (bamboo)

- Travis.yaml reverted back to original state.

### CODE REVIEW

**Comment by PunchedChimera** [comment]
Lgtm

File: src/main/java/org/alfresco/util/ConfigFileFinder.java [hunk 1]
```diff
@@ -112,6 +113,8 @@ public boolean readFiles(String path, Log log)
         {
             log.error("Error reading from "+path, e);
             successReadingConfig.set(false);
+        } catch (URISyntaxException e) {
+            e.printStackTrace();
```
Comment (ancutam) [inline src/main/java/org/alfresco/util/ConfigFileFinder.java]: Log the exception pls.

**Comment by KikoUK** [comment]
src/main/java/org/alfresco/util/ConfigFileFinder.java

File: src/main/java/org/alfresco/util/ConfigFileFinder.java [hunk 2]
```diff
@@ -108,7 +109,7 @@ public boolean readFiles(String path, Log log)
                 log.debug("No config read from "+path);
             }
         }
-        catch (IOException e)
+        catch (IOException | URISyntaxException e)
```
Comment (KikoUK) [inline src/main/java/org/alfresco/util/ConfigFileFinder.java:112]: Added URI exception to the existing catch clause as per @ancutam 's request.

**Comment by ancutam** [comment]
Look good, please squash and merge.

**Pull request merged**